### PR TITLE
Use longs for file offsets in stress client io

### PR DESCRIPTION
**Pull request by gpang**
Using longs supports larger file sizes.

### CODE REVIEW

File: stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java [hunk 1]
```diff
@@ -377,7 +377,7 @@ private int applyOperation() throws IOException {
           mInStream = mFs.open(mFilePath);
         }
         if (mParameters.mReadRandom) {
-          mCurrentOffset = mRandom.nextInt(mMaxOffset);
+          mCurrentOffset = Math.abs(mRandom.nextLong()) % mMaxOffset;
```
Comment (ZacBlanco) [inline stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java:380]: interesting. Is there no `Random#nextLong(long)` API?

Comment (gpang) [reply stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java]: Yeah, there isn't. However, I updated this to use the long stream.

**Comment by ZacBlanco** [comment]
I think the PR is fine as it is, but do you think it would be worth adding a seed parameter so that we can make reproducible tests? or run equivalent workloads?

**Comment by gpang** [comment]
Sure, that can come in a future PR.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**